### PR TITLE
Update sms-gateway.md

### PR DESCRIPTION
**Pull request by aaguiarz**
Added new fields to SMS Gateway and documented error handling.

### CODE REVIEW

**Comment by khorne3** [comment]
@aaguiarz Hey, I didn't notice at first this is still a draft -- should I leave this be for now?

File: articles/connections/passwordless/sms-gateway.md [hunk 1]
```diff
@@ -77,14 +78,43 @@ You can then send the updated configuration to the Management API using the [PAT
...
+
+If you set the `forward_req_info` property to **true**, the gateway will also receve information from the HTTP request that initiated the Passwordless process.
+
 ```
 {
   "recipient": "+1 399 999",
   "body": "Your verification code is: 12345",
   "sender": "+1 234 567",
+  "req" : { 
+      "ip" : "167.56.227.117",
```
Comment (thomasvandoren) [inline articles/connections/passwordless/sms-gateway.md:99]: How will this value be calculated in Auth0 tenant?

File: articles/connections/passwordless/sms-gateway.md [hunk 2]
```diff
@@ -77,14 +78,43 @@ You can then send the updated configuration to the Management API using the [PAT
...
+      "user-agent" : "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.109 Safari/537.36" 
+       }
 }
 ```
 
+## Error handling
+
+If the SMS Gateway returns an HTTP code other than 200, the `/passwordless/start` endpoint will return an HTTP 400 code and a response the looks like the following:
```
Comment (thomasvandoren) [inline articles/connections/passwordless/sms-gateway.md:109]: Are there any requirements for response body or content type from SMS Gateway?

File: articles/connections/passwordless/sms-gateway.md (hunk 1 above)
Comment (aaguiarz) [reply articles/connections/passwordless/sms-gateway.md:99]: It's the IP of the client calling /passwordless/start

File: articles/connections/passwordless/sms-gateway.md (hunk 2 above)
Comment (aaguiarz) [reply articles/connections/passwordless/sms-gateway.md:109]: No, we'll only consider the HTTP code

**Review by aaguiarz — COMMENTED** [review]
Responded and updated docs

File: articles/connections/passwordless/sms-gateway.md (hunk 1 above)
Comment (thomasvandoren) [reply articles/connections/passwordless/sms-gateway.md:99]: How does the tenant determine what the client IP is? Does it use x-forwarded-for when present, and if so does that mean any client can override client IP or change from call to call?

**Comment by khorne3** [comment]
@aaguiarz Adding a `do-not-merge` label to this. Please feel free to change when ready.

**Comment by khorne3** [comment]
@aaguiarz Can you resolve the merge conflicts? Thanks

**Review by khorne3 — APPROVED** [review]

**Pull request merged**